sql_scripts/prototype.sql

```diff
@@ -0,0 +1,34 @@
+drop table if exists instruments cascade;
+
+create table instruments as
+  select distinct(instrument) as name from points_prototype;
+
+alter table instruments
+   add column id INT generated always as identity,
+   add primary key(id);
+
+select * from instruments
+
+alter table points_prototype 
+  add instrument_id INT,
+  add constraint fk_instrument
+  foreign key (instrument_id)
+  references instruments (id);
+  
+insert into points_prototype (instrument_id)
```

Comment:
I think this needs to be an `update` as opposed to `insert` since we want to update the existing records

Reply:
Maybe something like this should work:
```sql
update points_prototype
set points_prototype.instrument_id = instrument.id
from instruments
where points_prototype.instrument = instruments.name
```

Reply:
I was thinking of that as a one-time activity to create the relation, populating a completely empty column. I am probably missing something?

Reply:
Yes, if you want to populate a newly added column to existing rows, then you need to use an `update`